backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name=Backend
```

Comment:
Prefer yml format.

I recommend to rename the file from application.properties to application.yml